ddl/failtest/fail_db_test.go

```diff
@@ -127,6 +127,10 @@ func (s *testFailDBSuite) TestHalfwayCancelOperations(c *C) {
 	tk.MustExec("insert into tx values(1)")
 	_, err = tk.Exec("rename table tx to ty")
 	c.Assert(err, NotNil)
+	tk.MustExec("create table ty(a int)")
+	tk.MustExec("insert into ty values(1)")
+	_, err = tk.Exec("rename table tx to ty, ty to tz")
+	c.Assert(err, NotNil)
```

Comment:
What if renaming `tx to ty` succeeds but `ty to tz` fails? Is `tx to ty` rolled back?

Reply:
They are in the same job. `tx to ty` should be rolled back. I am not sure whether I am right.

Reply:
I have add this case in the unit test, PTAL when you are free.